baseapp/baseapp.go

```diff
@@ -165,11 +161,6 @@ func (app *BaseApp) Name() string {
 	return app.name
 }
 
-// AppVersion returns the application's protocol version.
-func (app *BaseApp) AppVersion() uint64 {
-	return app.appVersion
-}
-
```

Comment:
What happens to code that calls this method?

Reply:
they would need to get the consensus params and see what's in there

Reply:
> they would need to get the consensus params and see what's in there

So this PR is a breaking change. Is that fact, or your proposed fix, documented anywhere? I couldn't find it in the diff for the PR, but I might not be looking in the right place.

Reply:
And is it actually true that `appVersion` _should_ be a part of consensus parameters? Is it not possible that an application updates its version in a way which is consensus-compatible?

Reply:
This will land with documentation, just want to test it first. I haven't had time to test it yet. appVersion is part of the consensus parameters from tendermint, we were not setting it on restart.